coopr-server/pom.xml

```diff
@@ -82,6 +82,14 @@
             </exclusion>
           </exclusions>
         </dependency>
+
+      <!-- REMOVE -->
+      <dependency>
+        <groupId>mysql</groupId>
+        <artifactId>mysql-connector-java</artifactId>
+        <version>5.1.34</version>
+      </dependency>
```

Comment:
This assumes a specific version though. If they are upgrading, it means they should already have everything set up with the jdbc driver.